build/teamcity-post-failures.py

```diff
@@ -85,7 +85,14 @@ def post_issue(issue):
         'https://api.github.com/repos/cockroachdb/cockroach/issues',
         data=json.dumps(issue).encode('utf-8'),
         headers={'Authorization': 'token {0}'.format(os.environ['GITHUB_API_TOKEN'])})
-    opener.open(req).read()
+    try:
+        opener.open(req).read()
+    except urllib.error.HTTPError as e:
+        # Github's error response bodies include useful information, so print them.
```

Comment:
Hope this doesn't leak our API token.

Reply:
No, that's in the header, not in the body. A sample error response looks like this according to github's docs:

```
HTTP/1.1 422 Unprocessable Entity
Content-Length: 149

{
  "message": "Validation Failed",
  "errors": [
    {
      "resource": "Issue",
      "field": "title",
      "code": "missing_field"
    }
  ]
}
```